### DISCUSSION TITLE
Playlists

### DISCUSSION THREAD

**Discussion by dandydon414** [strawberrymusicplayer/strawberry#1022] [Q&A]
Is there or will there be a feature to import more than 1 playlist?  thanks.

**Comment by ReIaxo** [comment]
There is a menu entry "Playlist -> Add directory..." (Have german version installed, so english naming is only a guess). In german version it is called "Playliste -> Ordner hinzufügen ..."
Nevertheless, when i click on it, open directory dialog opens. When I select a directory and click open, nothing happens. I think i will file an Issue for that.
